redbeams-api/src/main/java/com/sequenceiq/redbeams/doc/OperationDescriptions.java

```diff
@@ -26,14 +26,21 @@
 
     public static final class DatabaseOpDescription {
         public static final String POST_CONNECTION_TEST = "test RDS connectivity";
```

Comment:
Replace "RDS" with "database" for these descriptions.

Reply:
Done